jaeger-client/build.gradle

```diff
@@ -7,3 +9,28 @@ dependencies {
 
     testCompile group: 'junit', name: 'junit', version: junitVersion
 }
+
+/* Shaded the same libraries as jaeger-thrift,
+ * leaving the slf4j dependency in place, as a *requirement*
```

Comment:
This comment seems out of date now.

Reply:
Although not sure whether the slf4j-api should be shaded, as there is no slf4j logging impl as part of this uber jar, so would need to pick it up from the app's dependencies. Not sure what the impact of shading the api would have on picking up an impl?

Reply:
Yeah, I was wondering about this myself later on (whether shading it or not). So I think I will update the code to *not* shade it at all.